tests/wpt/web-platform-tests/html/semantics/forms/the-input-element/type-change-state.html

```diff
@@ -36,26 +36,34 @@
       if (types[i] != types[j]) {
         test(function() {
           var input = document.createElement("input");
+          var expected = "  foo\rbar  ";
           input.type = types[i].type;
           if (types[i].type === "file") {
             assert_throws("INVALID_STATE_ERR", function() {
-              input.value = "  foo\rbar  ";
+              input.value = expected;
```

Comment:
why set `input.value = expected;` here?
I think keep `expected = "  foo\rbar  "` is fine.

Reply:
That's fine, it makes it clearer what the expected value is for the tests.

Reply:
Moving `input.value = "  foo\rbar  "` is good.
But `input.value = expected;` seems unnecessary.

Reply:
I'm not following. This subtest in particular throws an exception when `input.value` is being assigned to. Are you saying that we should rather duplicate writing `"  foo\rbar  "` here?

Reply:
NO, we have moved `input.value = " foo\rbar "` to the front of `if` section.
`input.value` is set, and not need to set  `input.value = expected;`

Reply:
You've lost me. The change that this patch has is `var expected = "  foo\rbar  ";` instead of `input.value = "  foo\rbar  ";`